solr/contrib/ltr/src/java/org/apache/solr/ltr/FeatureLogger.java

```diff
@@ -36,46 +39,42 @@ protected FeatureLogger(String fvCacheName, FeatureFormat f) {
   }
 
   /**
-   * Log will be called every time that the model generates the feature values
-   * for a document and a query.
+   * Log will be called every time that the model generates the feature values for a document and a
+   * query.
    *
-   * @param docid
-   *          Solr document id whose features we are saving
-   * @param featuresInfo
-   *          List of all the {@link LTRScoringQuery.FeatureInfo} objects which contain name and value
-   *          for all the features triggered by the result set
-   * @return true if the logger successfully logged the features, false
-   *         otherwise.
+   * @param docid Solr document id whose features we are saving
+   * @param featuresInfo List of all the {@link LTRScoringQuery.FeatureInfo} objects which contain
+   *     name and value for all the features triggered by the result set
+   * @return true if the logger successfully logged the features, false otherwise.
    */
-
-  public boolean log(int docid, LTRScoringQuery scoringQuery,
-      SolrIndexSearcher searcher, LTRScoringQuery.FeatureInfo[] featuresInfo) {
+  public boolean log(
+      int docid,
+      LTRScoringQuery scoringQuery,
+      SolrIndexSearcher searcher,
+      LTRScoringQuery.FeatureInfo[] featuresInfo) {
     final String featureVector = makeFeatureVector(featuresInfo);
     if (featureVector == null) {
       return false;
     }
 
-    return searcher.cacheInsert(fvCacheName,
-        fvCacheKey(scoringQuery, docid), featureVector) != null;
+    return searcher.cacheInsert(fvCacheName, fvCacheKey(scoringQuery, docid), featureVector)
+        != null;
```

Comment:
I don't like this

Reply:
> I don't like this

how about https://github.com/apache/solr/pull/189/commits/4d13b97e41c72381367c37a8799a21169c8e0fa3 as a replacement?